assets/js/components/Landing.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import { Col, Row, Button } from 'reactstrap'
+import { connect } from 'react-redux'
+import { FormattedMessage } from 'react-intl'
+import isEmpty from 'lodash/isEmpty'
+
+import { fetchAllServices } from '../services/redux'
+import HelIcon from './HelIcon'
+import Service from './Service'
+
+const buttonWithIconStyle = {
+  margin: '2rem 0',
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'center'
+}
+
+class Landing extends Component {
+  componentDidMount() {
+    this.props.dispatch(fetchAllServices())
```

Comment:
If you add fetchAllServices to the connect part at the end of the file than I recall you can just call fetchAllServices() without the need for this.props.dispatch ?